### PR TITLE
initial project for link task

### PR DESCRIPTION
**Pull request by Ivan9009**
Add a link task in project.

### CODE REVIEW

File: webtasks-ivan/.idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mgenov) [inline webtasks-ivan/.idea/modules.xml]: You have to add idea project files to .gitignore.

Comment (Ivan9009) [inline webtasks-ivan/.idea/modules.xml]: Files are added to the project .gitignore.

**Comment by mgenov** [comment]
You have to squash your commits.

**Comment by Ivan9009** [comment]
Squash my commit in one.

**Pull request merged**